README.md

```diff
@@ -1,6 +1,8 @@
 # Doctrine Behavioral Extensions
 
-[![Build Status](https://travis-ci.org/Atlantic18/DoctrineExtensions.svg?branch=main)](https://travis-ci.org/Atlantic18/DoctrineExtensions)
+[![Continuous Integration](https://github.com/doctrine-extensions/DoctrineExtensions/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/doctrine-extensions/DoctrineExtensions/actions/workflows/continuous-integration.yml)
+[![Quality assurance](https://github.com/doctrine-extensions/DoctrineExtensions/actions/workflows/qa.yml/badge.svg)](https://github.com/doctrine-extensions/DoctrineExtensions/actions/workflows/qa.yml)
```

Comment:
```suggestion
[![Quality Assurance](https://github.com/doctrine-extensions/DoctrineExtensions/actions/workflows/qa.yml/badge.svg)](https://github.com/doctrine-extensions/DoctrineExtensions/actions/workflows/qa.yml)
```

Think we need to wrap them in a link? 🤔

Reply:
They are already wrapped by markdown anchor. Look at [this example](https://github.com/doctrine-extensions/DoctrineExtensions/blob/96d2fff9d4b6ceec24ba1a19497b3eb06396ea90/README.md#doctrine-behavioral-extensions).
About the "Quality Assurance" title, it is obtained from the action name, so I think we should change the name there: https://github.com/doctrine-extensions/DoctrineExtensions/actions/workflows/qa.yml.

Reply:
The replacement of "Quality assurance" by "Quality Assurance" was done (here and in the [related action](https://github.com/doctrine-extensions/DoctrineExtensions/actions/workflows/qa.yml)).